commands/addvertical.js

```diff
@@ -149,6 +197,15 @@ class VerticalAdder {
     if (args.cardName && !availableCards.includes(args.cardName)) {
       throw new UserError(`${args.cardName} is not a valid card`);
     }
+
+    if (args.locales.length) {
+      const supportedLocales = VerticalAdder._getAdditionalPageLocales(this.config);
+      args.locales.forEach(locale => {
+        if (!supportedLocales.includes(locale)) {
+          throw new UserError(`${locale} is not supported by your site`);
```

Comment:
nit: I think it might be helpful to include the fact that it's a locale which isn't supported. What do you think of updating the message to `${locale} is not a locale supported by your site`?